src/components/editTestCase/qiCore/LeftPanel/ElementsTab/Demographics/DemographicsSection.tsx

```diff
@@ -141,6 +167,19 @@ const DemographicsSection = ({ canEdit }) => {
           <div className="demographics-container">
             <div className="demographics-row">
               <FormControl tw={"w-2/4"}>
+                <DateField
+                  label="Date of Birth"
+                  disabled={!canEdit}
+                  value={patient?.birthDate ? dayjs(patient.birthDate) : null}
+                  handleDateChange={(event: any) => {
+                    const date = new Date(event);
```

Comment:
creating a date from an "event" seems a bit confusing. I don’t think it is an actual event object. naming it "value" or something other than event would be nice.